### PR TITLE
DIV-3773 Added chart config

### PR DESCRIPTION
**Pull request by qzhou-hmcts**
# Description

• AKS configs have moved from ` src/kubernetes/` to ` charts/div-{project}`
•  If you ever logged into the K8S dashboard, PR containers now live in the “divorce” namespace, as opposed to their own `div-pr-{id}` namespace
• There is a new liveness endpoint to check if the pod is up (regardless of dependencies) so that pods don’t keep restarting if a dependency is not 100% stable

Fixes # (issue)

[DIV-3773](https://tools.hmcts.net/jira/browse/DIV-3773)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tchow8** [comment]
@qzhou-hmcts @hosainnet FE modifications for the properties volume ready for review.
@dtaylor7 please check the setupSecrets middleware and let me know what you think. Since AAT will still be using env vars, whilst Helm will be using the new properties volume to populate secrets.

File: charts/div-pfe/values.aat.template.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        PAYMENT_REFERENCE_SERVICE_IDENTIFICATION: "DIV1"
+        CASE_ORCHESTRATION_SERVICE_URL: "http://div-cos-aat.service.core-compute-aat.internal"
+        CASE_ORCHESTRATION_SERVICE_DRAFT_URL: "http://div-cos-aat.service.core-compute-aat.internal/draftsapi/version/1"
+        CASE_ORCHESTRATION_SERVICE_HEALTHCHECK_URL: "http://div-cos-aat.service.core-compute-aat.internal/health"
+        SERVICE_AUTH_PROVIDER_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+        SERVICE_AUTH_PROVIDER_HEALTHCHECK_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal/health"
+        IDAM_LOGIN_URL: "https://idam.preprod.ccidam.reform.hmcts.net/login"
+        IDAM_AUTHENTICATION_HEALHCHECK_URL: "https://idam.preprod.ccidam.reform.hmcts.net/health"
+        IDAM_APP_HEALHCHECK_URL: "https://preprod-idamapi.reform.hmcts.net:3511/health"
+        IDAM_API_URL: "https://preprod-idamapi.reform.hmcts.net:3511"
```
Comment (qzhou-hmcts) [inline charts/div-pfe/values.aat.template.yaml]: @tchow8  probably wants to use sidam url

Comment (tchow8) [reply charts/div-pfe/values.aat.template.yaml]: Changed

**Comment by hosainnet** [comment]
@qzhou-hmcts conflicts, FYI

**Comment by tchow8** [comment]
Auto Whitelisting Proved by:
https://github.com/hmcts/div-petitioner-frontend/pull/512

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -6,7 +6,7 @@ String product = "div"
 String component = "pfe"
 
 List<LinkedHashMap<String, Object>> secrets = [
-  secret('session-secret', 'SESSION_SECRET'),
+  secret('session-secret', 'SECRET'),
```
Comment (hosainnet) [inline Jenkinsfile_CNP]: Did you mean to rename this to just "SECRET"?

File: charts/div-pfe/values.aat.template.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        CASE_ORCHESTRATION_SERVICE_URL: "http://div-cos-aat.service.core-compute-aat.internal"
+        DECREE_NISI_FRONTEND_URL: "https://div-dn-aat.service.core-compute-aat.internal"
+        DEPLOYMENT_ENV: "preprod"
+        EVIDENCE_MANAGEMENT_CLIENT_API_HEALTHCHECK_URL: "http://div-emca-aat.service.core-compute-aat.internal/health"
+        EVIDENCE_MANAGEMENT_CLIENT_API_URL: "http://div-emca-aat.service.core-compute-aat.internal"
+        FEATURE_IDAM: "true"
+        FEATURE_RELEASE_510: "true"
+        FEATURE_RELEASE_520: "false"
+        FEATURE_RELEASE_520_DESERTION: "true"
+        FEATURE_RELEASE_530: "false"
```
Comment (hosainnet) [inline charts/div-pfe/values.aat.template.yaml]: I think these flags are out of date now..

File: charts/div-pfe/values.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+nodejs:
+    applicationPort: 8080
+    environment:
+        REFORM_SERVICE_NAME: "pfs"
+        REFORM_TEAM: "div"
+        EVIDENCE_MANAGEMENT_CLIENT_API_UPLOAD_ENDPOINT: "/emclientapi/version/1/upload"
+        FEATURE_IDAM: "true"
+        FEATURE_REDIRECT_TO_APPLICATION_SUBMITTED: "false"
+        FEATURE_RELEASE_510: "true"
+        FEATURE_RELEASE_520: "false"
```
Comment (hosainnet) [inline charts/div-pfe/values.yaml]: Defaults flags should be updated here

File: charts/div-pfe/values.aat.template.yaml (hunk 3 above)
Comment (hosainnet) [reply charts/div-pfe/values.aat.template.yaml]: Maybe if we update the flags below, we can get rid of these

File: charts/div-pfe/values.preview.template.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        CASE_ORCHESTRATION_SERVICE_URL: "http://div-cos-aat.service.core-compute-aat.internal"
+        DECREE_NISI_FRONTEND_URL: "https://div-dn-aat.service.core-compute-aat.internal"
+        DEPLOYMENT_ENV: "preprod"
+        EVIDENCE_MANAGEMENT_CLIENT_API_HEALTHCHECK_URL: "http://div-emca-aat.service.core-compute-aat.internal/health"
+        EVIDENCE_MANAGEMENT_CLIENT_API_URL: "http://div-emca-aat.service.core-compute-aat.internal"
+        FEATURE_IDAM: "true"
+        FEATURE_RELEASE_510: "true"
+        FEATURE_RELEASE_520: "true"
+        FEATURE_RELEASE_520_DESERTION: "true"
+        FEATURE_RELEASE_530: "true"
```
Comment (hosainnet) [inline charts/div-pfe/values.preview.template.yaml]: And these

File: Jenkinsfile_CNP (hunk 2 above)
Comment (tchow8) [reply Jenkinsfile_CNP]: Yes, matching https://github.com/hmcts/div-petitioner-frontend/blob/master/config/custom-environment-variables.yaml#L38

File: charts/div-pfe/values.aat.template.yaml (hunk 3 above)
Comment (tchow8) [reply charts/div-pfe/values.aat.template.yaml]: Just in case the feature flags differ per env, I'm leaving it in here and removing it from the default.

**Comment by hosainnet** [comment]
@dtaylor7 can you please re-review?

**Pull request merged**